runtime/autoload/dist/ft.vim

```diff
@@ -877,6 +877,23 @@ func dist#ft#FTsig()
   endif
 endfunc
 
+" This function checks the first 100 lines of files matching "*.sil" to
+" resolve detection between Swift Intermediate Language and SILE.
+func dist#ft#FTsil()
+  for lnum in range(1, min([line('$'), 100]))
```

Comment:
```suggestion
  for lnum in range(1, [line('$'), 100]->min())
```

Reply:
Oh, I wasn't sure if that's Vim9-only syntax so I used the same approach as in existing detection functions

Reply:
This is the only place where `->min()` is used in Vim as well.